src/pages/index.tsx

```diff
@@ -338,17 +450,19 @@ const Home: NextPage<HomePageProps> = ({ marketplace }) => {
               className="absolute border-4 object-cover border-gray-900 bg-gray-900 rounded-full w-28 h-28 -top-32"
             />
             <h1>{marketplace.name}</h1>
-            <p className="mt-4 max-w-prose">{marketplace.description}</p>
+            <p className="mt-4 max-w-prose text-gray-300">
```

Comment:
Need a max width on the p noticed it stretches all the way to the stats block when the description is longer.

Reply:
Sure, providing the required right margin for this.